src/main/scala/curlrequests/package.scala

```diff
@@ -46,12 +47,18 @@ package object CurlRequests {
   def parseJsonFromRequest(json: JsValue) : List[String] = {
     val list = json \ "hits" \ "hits"
 
-    list.as[List[JsObject]].map { x =>
-      (x \ "_source" \ "log").validate[String].map { y =>
-          val some = play.api.libs.json.Json.parse(y).as[JsonClass]
-          some.message
-      }.get
-    }
+    list.validate[List[JsObject]].map {
```

Comment:
`.map` over `JsResult` maps only over `JsSuccess`, you should do pattern matching instead. Goal is to get rid of `.get` which is also unsafe.